sim/env.py

```diff
@@ -3,9 +3,16 @@
 import os
 from pathlib import Path
 
+from dotenv import load_dotenv
+
+load_dotenv()
+
 
 def model_dir() -> Path:
-    return Path(os.environ.get("MODEL_DIR", "models"))
+    # return Path(os.environ.get("MODEL_DIR", "models"))
+    # return Path(os.environ.get("MODEL_DIR", "stompytherobot"))
+    # return Path("/home/dpsh/isaacs_sim/sim/sim/stompy2")
+    return Path("/home/dpsh/isaacs_sim/sim/sim/stompy2")
```

Comment:
export MODEL_DIR=/home/dpsh/isaacs_sim/sim/sim/stompy2 in terminal would do the job

Reply:
done